### DISCUSSION TITLE
Unexpected imaginary frequencies during the SCPH calculation

### DISCUSSION THREAD

**Discussion by Lamed-git** [ttadano/alamode#185] [Q&A]
Dear Dr. Tadano and users,
 When I calculated a tetragonal crystal system, if I use a supercell (2 2 1, 64 atoms) derived from the conventional cell to fit the force constants, the harmonic phonon spectrum shows no imaginary frequencies. However, when performing SCPH calculations using the primitive cell, the calculation fails, indicating a lack of consistency between the supercell and primitive cell. Therefore, I switched to using a supercell (2 2 2, 64 atoms) derived from the primitive cell to fit the force constants. However, two issues arose: firstly, imaginary frequencies appeared near the Gamma point in the harmonic phonon spectrum. 
![image](https://github.com/ttadano/alamode/assets/81789705/63ea5498-d35e-4b6e-a96a-d34b2442b360)
I attempted to resolve this by using a larger supercell (3 3 3, 216 atoms) to no avail. Hence, I reduced the cutoff value for the second-order force constants from None to 18 Bohr (considering that this might be due to insufficient description of long-range interactions), which eliminated the imaginary frequencies. I wonder if this operation is reasonable.
Secondly, however, when considering the Bubble process in the SCPH calculation (supercell of 216 atoms, 18 Bohr cutoff for harmonic force constants), imaginary frequencies also appear at the Gamma point in the renormalized phonon spectrum. I'm not entirely sure about the source of these imaginary frequencies and how to eliminate them.
![image](https://github.com/ttadano/alamode/assets/81789705/77af487f-11cb-4119-b1dc-9a674f5b3fb1)

**Comment by zhanglin-alt** [comment]
Hello, I was wondering if you have identified the cause of the imaginary frequencies in the bubble calculation. Recently, I have been struggling with this issue: my self-consistent loop does not show any imaginary frequencies, but the bubble calculation does. Do you have any insights into why this discrepancy occurs? I'm not sure whether it's due to the third-order cutoff or the KMESH being too small to achieve convergence.

**Reply by ttadano** [reply]
The bubble correction can make phonons unstable unlike the SCP (without bubble).

**Reply by zhanglin-alt** [reply]
Dear Dr. Tadano，
Thank you for your reply. However, my system has been experimentally and theoretically (via tdep) confirmed to be stable at 300 K. The training dataset is from the AIMD simulation ，a NVT ensemble with the Nosé Hoover thermostat at 300 K was ran for 1000-time steps using a 1-fs time step. By disregarding the first 100 steps from the obtained AIMD trajectories, 160 atomic structures were sampled at equal intervals. Therefore, I am unsure whether the issue lies in my input parameters or the training dataset. Below are my key optimization settings：
&general
  PREFIX = LYTO222
  MODE = opt
  TOLERANCE = 1.0E-3   # Tolerance for finding symmetry operations，Default:1.0e-3
  NAT = 224;  # Number of atoms in the supercell
  NKD = 4  # Number of atomic species
  KD = Li Y Ti O
/

&interaction
  NORDER = 3  # (Number of order)-1, 1: harmonic, 2: cubic, ..
  NBODY = 2 3 3
/

&cutoff
  *-* none 10 8
/

&optimize
    LMODEL = enet
    cv = 0
    L1_ALPHA = 1e-10
    L1_RATIO =  1   
    STANDARDIZE = 1         
    MAXITER = 500    
    CONV_TOL = 1.0e-10      
    DEBIAS_OLS = 1     
    STOP_CRITERION = 5   
    FC2XML = fc2.xml
    DFSET = disp_force.dat

**Reply by vivekgusain09** [reply]
<img width="1378" height="905" alt="image" src="https://github.com/user-attachments/assets/b40eea6d-0038-4763-8fcb-ecbbfd1910f1" />
Hi all. I am also facing the same issue, the scp bands does not show any imaginary frequencies however when computed the scp+bubble phonon dispersions there are some strange kinks in the acoustic modes near zero frequencies. Can anyone help me with that if i am making some mistakes in the calculations since the k mesh is also reasonable (8 x 8 x 8). (I did got the warning the warning about the acoustic sum rule broken)

**Reply by vivekgusain09** [reply]
@ttadano  @zhanglin-alt  Were you able to determine the reason/solution to these unusual kinks in acoustic modes that lead to the imaginary phonon? Could you please help me understand this as I am having the same issue? I tried my best by performing various test calculations. Also, can I trust the lattice thermal conductivity calculations performed using these renormalized phonon frequencies?
